client/App.js

```diff
@@ -1,10 +1,19 @@
 import React, { useState } from 'react'
 import Hello from './Components/Hello'
+import Calendar from 'react-calendar'
+import CalendarRawCss from 'react-calendar/dist/Calendar.css?raw'
 
 const AppComponent = () => {
   const [name] = useState('world')
 
-  return <Hello name={name} />
+  return (
+    <>
+      <Hello name={name} />
+      <div css={CalendarRawCss}>
+        <Calendar />
+      </div>
```

Comment:
wrapper you component to a div element and pass raw css content to css attribute.